configuration/using-dashboard.rst

```diff
@@ -6,10 +6,10 @@ Hangfire Dashboard is a place where you could find all the information about you
 .. contents::
    :local:
 
-Adding Dashboard
+Adding Dashboard (OWIN)
 -----------------
 
-.. admonition:: Additional package required for ASP.NET + IIS
+.. admonition:: Additional package required for `ASP.NET <getting-started/aspnet-applications.rst>`_ and `ASP.NET Core <getting-started/aspnet-core-applications.rst>`_
```

Comment:
Hm, for ASP.NET that additional package is Microsoft.Owin.Hosting.SystemWeb which is outside of Hangfire ecosystem. And it's unclear what is the additional package for ASP.NET Core. I suppose it's `Hangfire.AspNetCore`, but in this section we are talking solely about OWIN environment as you highlighted in the line 9. I believe it's better to remove change in this line.

Reply:
@odinserj I just took another peak at the "Getting Start ASP.NET" and now I think it makes sense. 

I think its a bit weird to have this whole section on "adding dashboard" but only for "owin & asp.net", meanwhile each getting started contains samples for adding dashboard overlapping.

Do you think it would make sense to introduce a ASP.NET Core section here as well and possibly leaving that in getting started?

Reply:
> I think its a bit weird to have this whole section on "adding dashboard" but only for "owin & asp.net", meanwhile each getting started contains samples for adding dashboard overlapping.

Yes, both adding and configuring dashboard sections are for ASP.NET applications only, and ASP.NET Core-related information wasn't written. 

> Do you think it would make sense to introduce a ASP.NET Core section here as well or leaving that in getting started?

Who knows, documentation articles are always denormalized, i.e. the same information is duplicated in different articles, and the main difficulty is to keep everything in-sync. So I believe there's no good answer to this question that's both making it easy to find information and easy to maintain. But for docs the former one is more important IMHO.